quotes/dyvue.yaml

```diff
@@ -0,0 +1,11 @@
+username: dyvue
+quotes:
+  - text: 'Better than yesterday worse than tomorrow'
+    author: 'Romadhan Edy'
+    created_at: '5 Oktober 2021'
+  - text: 'Semua orang itu punya privilige'
```

Comment:
Awesome @dyvue , before we merge, can you fix this typo please. It's should be `privilege`.

Reply:
Yes, I fixed it, thanks anyway

Reply:
Thanks @dyvue